### ISSUE TITLE
Feature request: Retweet/Quote tweets with a seperate tab without filtering from following tab

### ISSUE THREAD

**Issue by Enduser-only** [insin/control-panel-for-twitter#343]
Under settings for retweets and quote tweets, each can be set to either do nothing, hide them, or show in a separate tab. Would it be possible to add an option where retweets and quote tweets are both in the following tab and the separate shared tab/retweets/quote tweets tab, without removing/filtering them from the following tab?

**Comment by insin** [comment]
Yes, should be easy, the hardest part will be coming up with a name for the options. Currently we have:

- Do nothing
- Hide
- Show in separate tab

Something like:

- Do nothing
- Hide
- Move to separate tab
- Also show in separate tab

"Also show in separate tab" is a bit clunky, so I'm open to any ideas 😼
